### PR TITLE
Migrate gentoo importer #1055

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: vulnerabilities/importers/gentoo.py [hunk 1]
```diff
@@ -10,73 +10,69 @@
...
+                yield from self.process_file(file_path)
+        finally:
+            if self.vcs_response:
+                self.vcs_response.delete()
 
     def process_file(self, file):
-        xml_data = {}
+        cves = []
+        summary = ""
+        vuln_reference = []
```
Comment (TG1999) [inline vulnerabilities/importers/gentoo.py:45]:
```suggestion
        vuln_references = []
```

**Review by TG1999 — COMMENTED** [review]
Suggestions

**Review by TG1999 — COMMENTED** [review]
Suggestion

**Review by pombredanne — APPROVED** [review]
There are a few changes to apply here to simplify the splits()
But otherwise, this is good to go.

**Pull request merged**